scripts/format.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+import multiprocessing
+import os
+import sys
+from functools import partial
+from subprocess import STDOUT, CalledProcessError, check_output
+
+from termcolor import cprint
+
+import b
+
+
+def _do_format(path):
+    text = ""
+    try:
+        # Use the absolute path to file
+        abs_path = os.path.abspath(path)
+
+        if path.endswith((".h", ".c", ".cc", ".cxx", ".cpp", ".hpp", ".ipp")):
+            text = "Formatting {} with clang-format\n".format(path)
+            text = text + check_output(["clang-format", "-i", "-style=file", abs_path], stderr=STDOUT).decode("utf-8")
+        elif path.endswith(("CMakeLists.txt", ".cmake")):
+            text = "Formatting {} with cmake-format\n".format(path)
+            text = text + check_output(["cmake-format", "-i", abs_path], stderr=STDOUT).decode("utf-8")
+        elif path.endswith((".py")):
+            text = "Formatting {} with isort and black\n".format(path)
+            text = text + check_output(["isort", abs_path], stderr=STDOUT).decode("utf-8")
+            text = text + check_output(["black", abs_path], stderr=STDOUT).decode("utf-8")
+    except CalledProcessError as e:
+        text = text + e.output.decode("utf-8")
+    except:
+        cprint("You don't have one of the formatters installed.\nRun `sudo -H pip3 install -r requirements.txt`", color='red')
+
+    return text
+
+
+def register(command):
+    command.help = "Format all the code in the codebase using clang-format"
+    print("nice kevin")
```

Comment:
Leave kevin alone!

Reply:
Is this a self reference to _Big Kevin_? 😅

Reply:
hahhahaaha anyone who can think of something that doesn't sound stupid for debug print statements is a psychopath